packages/core/test/acceptance/routing/routing.acceptance.ts

```diff
@@ -203,6 +208,32 @@ describe('Routing', () => {
     });
   });
 
+  it('supports function-based routes', async () => {
+    const app = givenAnApplication();
+
+    const routeSpec = <OperationObject> {
+      parameters: [
+        <ParameterObject> {name: 'name', in: 'query', type: 'string'},
+      ],
+      responses: {
+        '200': <ResponseObject> {
```

Comment:
Interesting. I thought that javascript objects distinguish between number and string property keys. Apparently not. I'll apply your suggestion.